dev/ToastNotifications/ToastActivationCallback.h

```diff
@@ -18,7 +18,7 @@ struct ToastActivationCallbackFactory : winrt::implements<ToastActivationCallbac
 {
     STDMETHODIMP CreateInstance(_In_opt_ IUnknown* aggregateInterface, _In_ REFIID interfaceId, _Outptr_ VOID** object) noexcept final try
     {
-        RETURN_HR_IF_NULL(CLASS_E_NOAGGREGATION, aggregateInterface);
+        RETURN_HR_IF(CLASS_E_NOAGGREGATION, aggregateInterface != nullptr);
```

Comment:
please help me understand this, is this when aggregateInterface is not null, we will return CLASS_E_NOAGGREGATION?

Reply:
Right, this bug was introduced in the last PR which would cause the ToastActivationCallback to be not able to be CoCreateInstance'd.